pythonforandroid/bootstraps/common/build/jni/application/src/start.c

```diff
@@ -356,7 +358,91 @@ int main(int argc, char *argv[]) {
   return ret;
 }
 
-JNIEXPORT void JNICALL Java_org_kivy_android_PythonService_nativeStart(
+JNIEXPORT int JNICALL Java_org_kivy_android_PythonService_nativeStart(
```

Comment:
Only thing that changes between `Java_org_kivy_android_PythonService_nativeStart()`, `Java_org_kivy_android_PythonWorker_nativeStart()` and `Java_org_kivy_android_PythonBoundService_nativeStart()` is the function name.
Maybe we should share the body via a commun function

Reply:
sure